crawler/src/main/java/com/brandwatch/minibcr/crawler/config/CronConfig.java

```diff
@@ -0,0 +1,9 @@
+package com.brandwatch.minibcr.crawler.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+@Configuration
+@EnableScheduling
+public class CronConfig {
```

Comment:
Interesting class. I probably would've just put `@EnableScheduling` on the class with the main method and `@SpringBootApplication`. This is not wrong, it just seems a bit overengineered I think. Unless you expect this class to grow?

Reply:
Yes, my thought was all configs at one package separated. and any related configuration addition will be easy for future.

Reply:
I continue to think it's a bit overengineered, but I don't have a big problem with it either.